src/scripts/main.js

```diff
@@ -1,5 +1,31 @@
 'use strict';
 
 document.addEventListener('click', e => {
-  // write code here
+  const spider = document.querySelector('.spider');
+  const wall = document.querySelector('.wall');
+  const halfWidth = spider.offsetWidth / 2;
+  const halfHeigth = spider.offsetHeight / 2;
+
+  if (e.target.className !== 'wall') {
+    return;
+  }
```

Comment:
What if I want to move spider a bit from current possition?
https://user-images.githubusercontent.com/61904995/225906750-bbe40917-0f0d-4408-8444-51172195ec30.mp4